internal/tools.go

```diff
@@ -27,4 +27,5 @@ import (
 	_ "github.com/jstemmer/go-junit-report"
 	_ "golang.org/x/lint/golint"
 	_ "golang.org/x/tools/cmd/goimports"
+	_ "github.com/client9/misspell/cmd/misspell"
```

Comment:
nit: didn't goimports complain about the order here? :)

Reply:
It did :) fixed in https://github.com/open-telemetry/opentelemetry-service/pull/214/commits/99eabe206252d78b0006b0380930e17fcc3e652a.

Reply:
:)